index.html

```diff
@@ -1,657 +1,1213 @@
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/start-a-group.html" class="cmp-navigation__item-link">Start a Group</a>
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/resources.html" class="cmp-navigation__item-link">Resources</a>
+                        </li>
+  
+                        <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                          <a href="/content/leaderimpact/lang/en2/stories.html" class="cmp-navigation__item-link">About</a>
+                        </li>
+                      </ul>
                     </nav>
-                </div>
-                <div class="sub-nav">
-                    <span class="sub-nav-item">
-                        <a data-cmp-clickable="" class="nav-link-item" href="">Give</a>
-                    </span>
-                    <div class="nav-button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Leaderhub</span>
+                  </div>
+                  <!-- SUB NAV -->
+                  <div class="container responsivegrid sub-nav">
+                    <div id="container-0b2a65237d" class="cmp-container">
+                      <!-- SUB NAV LINK -->
+                      <div class="link">
+                        <div id="text-1ffc132fa9" class="cmp-link">
+                          <a href="#">Give</a>
+                        </div>
+                      </div>
+                      <!-- SUB NAV BUTTON -->
+                      <div class="button nav-button">
+                        <a id="button-4e7f03c2b8" class="cmp-button" href="/content/leaderimpact/lang/en2/leaderhub.html">
+                          <span class="cmp-button__text">Leaderhub</span>
                         </a>
+                      </div>
+                      <!-- IMAGE SEARCH -->
+                      <div class="image search">
+                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/image.coreimg{.width}.svg/1669058888816/search.svg" data-asset="/content/dam/LeaderImpact/site-./assets/search.svg" data-title="search" id="image-fd5f155c3f" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                          <img src="./assets/search.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="search">
+                        </div>
+                      </div>
+                      <!-- HAMBURGER WRAP -->
+                      <div class="container responsivegrid hamburger">

```

Comment:
@Prystal254 Can we get rid of this wrapping container and use a mobile-only class on the buttons themselves?